api/config/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	DefaultSecretStorage *SecretStorage         `validate:"required"`
 	UI                   *UIConfig
 	Webhooks             *webhooks.Config
+	UpdateProject        *UpdateProjectConfig
```

Comment:
Can we rename both the name of this field and the name of this field's class? `UpdateProject` sounds like an action and it sounds very weird for a field name. Something like `UpdateProjectExtEndpoint` sounds clearer (it's a bit long but if you have other better ideas please suggest them!). For the class name we can rename it to something like `UpdateProjectExtEndpointConfig`.